### PR TITLE
Update ja_jp.json

### PR DESCRIPTION
**Pull request by Toshimichi0915**
Everything in en_us.json has been translated into Japanese.

### CODE REVIEW

File: enigma/src/main/resources/lang/ja_jp.json [hunk 1]
```diff
@@ -81,11 +85,14 @@
 	"popup_menu.editor_tab.close_right": "右のタブを全て閉じる",
 	"popup_menu.deobf_panel.rename_package": "パッケージをリネームする",
 	"popup_menu.deobf_panel.rename_class": "クラスをリネームする",
+	"popup_menu.deobf_panel.expand_all": "全て展開する",
+	"popup_menu.deobf_panel.collapse_all": "全て縮小する",
```
Comment (apple502j) [inline enigma/src/main/resources/lang/ja_jp.json:89]:
```suggestion
	"popup_menu.deobf_panel.collapse_all": "全て折りたたむ",
```

File: enigma/src/main/resources/lang/ja_jp.json [hunk 2]
```diff
@@ -46,6 +47,9 @@
 	"menu.view.change.title": "変更",
 	"menu.view.change.summary": "変更は次の起動時に適用されます",
 	"menu.search": "検索",
+	"menu.search.class": "クラスを検索する",
```
Comment (apple502j) [inline enigma/src/main/resources/lang/ja_jp.json:50]: Perhaps de-する these? These are menu items, which usually get 体言止めd

File: enigma/src/main/resources/lang/ja_jp.json [hunk 3]
```diff
@@ -106,6 +113,21 @@
 	"info_panel.tree.implementations": "実装",
 	"info_panel.tree.calls": "呼び出し関係",
 
+	"popup.copied": "コピーされました!",
```
Comment (apple502j) [inline enigma/src/main/resources/lang/ja_jp.json:116]:
```suggestion
	"popup.copied": "コピーしました!",
```

File: enigma/src/main/resources/lang/ja_jp.json [hunk 4]
```diff
@@ -106,6 +113,21 @@
...
+
+	"structure.options.obfuscation": "難読化の可視性",
```
Comment (apple502j) [inline enigma/src/main/resources/lang/ja_jp.json:118]: This does not change the 可視性 of 難読化, it only hides/shows based on 難読化, better name?

File: enigma/src/main/resources/lang/ja_jp.json [hunk 5]
```diff
@@ -106,6 +113,21 @@
...
+	"structure.options.obfuscation.all": "全て",
+	"structure.options.obfuscation.obfuscated": "難読化されたもののみ",
+	"structure.options.obfuscation.deobfuscated": "難読化解除されたもののみ",
+	"structure.options.documentation": "ドキュメントの可視性",
```
Comment (apple502j) [inline enigma/src/main/resources/lang/ja_jp.json:122]: Same goes here, this cannot hide javadoc

**Review by apple502j — CHANGES_REQUESTED** [review]

File: enigma/src/main/resources/lang/ja_jp.json (hunk 4 above)
Comment (Toshimichi0915) [reply enigma/src/main/resources/lang/ja_jp.json:118]: image

I don't think 難読化の可視性 is that much confusing. "Obfuscation Visibility" is really translated into "難読化の可視性."

**Comment by Toshimichi0915** [comment]
squash & merge please

**Comment by 2xsaiko** [comment]
Thanks!

> squash & merge please
FYI, you can do that yourself with `git rebase -i` (at least, that's one way to do it)

**Pull request merged**